#!/usr/bin/env python
PACKAGE = "lidar_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("range_min", double_t, 0, "Minimum range for filtering pointcloud point", 0.0, 0.0, 100.0)
gen.add("range_max", double_t, 0, "Maximum range for filtering pointcloud point", 30.0, 0.0, 100.0)

gen.add("fill_depth_image", bool_t, 0, "Fill the whole depth image", True)
gen.add("kernel_size", int_t, 0, "Size of the dilation kernel", 3, 1, 10)
gen.add("dilation_itr", int_t, 0, "Number of dilation itterations", 10, 1, 30)

exit(gen.generate(PACKAGE, "lidar_camera", "LidarCam"))